src/components/MetadataSidebar.vue

```diff
@@ -25,7 +25,8 @@
       </oc-button>
     </div>
     <div v-if="isMetadataExtracted" id="dicom-metadata-sidebar-content" class="oc-p-s">
-      <table class="details-table">
+      <p id="mydesc">DICOM metadata details table</p>
```

Comment:
what is your reasoning for adding the id? where is it used?

Reply:
it is used in the table element